### PR TITLE
Updating server/database data

### PR DESCRIPTION
**Pull request by AnthonyDario**
Updated the database so that it now stores all of the data (resources, Elemari) the front-end wants. The server will now return this data upon login. I also updated the SaveState script so that it stores the new data when it pulls from the server.

### CODE REVIEW

File: server/models/user.py [hunk 1]
```diff
@@ -18,4 +18,15 @@ class User(Base):
...
-
+    fire = Column(Integer, default = 0)
+    air = Column(Integer, default = 0)
+    water = Column(Integer, default = 0)
+    earth = Column(Integer, default = 0)
+    maxFire = Column(Integer, default = 10) 
+    maxAir = Column(Integer, default = 10) 
+    maxWater = Column(Integer, default = 10) 
+    maxEarth = Column(Integer, default = 10) 
+    fireEle = Column(Integer, default = 1)
```
Comment (CatalystOfNostalgia) [inline server/models/user.py]: Let's also add a poof count.

File: server/sql/create_tables.sql [hunk 2]
```diff
@@ -8,6 +8,18 @@ CREATE TABLE user(
 	email varchar(100) NOT NULL UNIQUE,
 	username varchar(100) NOT NULL UNIQUE,
 	password varchar(100),
+    fire int DEFAULT 0,
```
Comment (CatalystOfNostalgia) [inline server/sql/create_tables.sql:11]: These tabs don't seem to be aligned properly.

**Comment by AnthonyDario** [comment]
Added the poof count and fixed style. Some people are using spaces and some people are using tabs for indentation...

**Comment by CatalystOfNostalgia** [comment]
Merged @AnthonyDario

**Pull request merged**